app/models/poll.rb

```diff
@@ -27,7 +27,7 @@ class Poll < ActiveRecord::Base
   validate  :date_should_be_in_future
 
   after_create  :create_rating
-  after_find    :close_if_expire
+  after_find    :close_if_expire, :status_machine
```

Comment:
I'd prefer this to be called `initialize_status_machine` (but can live with the current name).
Isn't it possible to put this `after_find` initializer into the `PollStatusMachine` module somehow? This place seems to be quite error-prone: it's enough to rename the method in the module, or forget to add callback in your model — and you have malfunctioning code.